working/augmentation-libraries/feature-specification.md

```diff
@@ -278,6 +280,38 @@ It is a compile-time error if:
     original declaration occurs, according to merge order. *An augmentation
     library can both declare a new declaration and augment it in the same file.*
 
+### Augment super
+
+The exact result of an `augment super` expression depends on what is being
+augmented, but it follows generally the same rules as any normal identifier:
+
+*   **Augmenting getters**: Within an augmenting getter `augment super` invokes
+    the getter and evaluates to the return value. If augmenting a field with a
+    getter, this will invoke the implicit getter from the augmented field.
+
+*   **Augmenting setters**: Within an augmenting setter `augment super` must be
+    followed by an `=` and will directly invoke the augmented setter. If
+    augmenting a field with a setter, this will invoke the implicit setter from
+    the augmented field.
+
+*   **Augmenting fields**: Within an augmenting field, `augment super` can only
+    be used in an initializer expression, and refers to the original fields
+    initializer expression, which is immediately evaluated. If there is no
+    initializer expression (or augmenting a getter and/or setter), then
+    `augment super` is not allowed.
```

Comment:
Instead of "is not allowed", I would be more precise:

> It is a compile-time error to use `augment super` in an augmenting field's initializer if the member being augmented is not a field with an initializer.

Also, we might need to be clearer about how this works with chained augmentations. If you:

1. A field declaration.
2. Augmenting that with a getter.
3. Augmenting that with a field.

Can the initializer in 3 use `augment super` to invoke the initializer from 1?

Reply:
> Can the initializer in 3 use `augment super` to invoke the initializer from 1?

Haha, nothing is ever easy is it?

Reply:
I added a TODO here, this brings up a whole can of worms issue.